README.md

```diff
@@ -191,6 +191,13 @@ spec:
   type: LoadBalancer
 ```
 
+## Tips
+Domain Mapping is configured to explicitly use `http2` protocol only. This behaviour can be disabled by adding the following annotation to the Domain Mapping resource
+```
+kubectl annotate domainmapping <domain_mapping_name> networking.knative.dev/disableHTTP2=true --namespace <namespace>
```

Comment:
Could you please change the annotation as:
```suggestion
kubectl annotate domainmapping <domain_mapping_name> kourier.knative.dev/disable-http2=true --namespace <namespace>
```

Because this annotation works only for Kourier not Istio/Contour for now.